radar.csv

```diff
@@ -13,7 +13,7 @@ name,ring,quadrant,isNew,description
 "WinForms",hold,"Languages and Frameworks",true,"Don't use WinForms for new projects - prefer Electron."
 "WPF",hold,"Languages and Frameworks",true,"Don't use WPF for new projects - prefer Electron."
 "Object–relational mappers (ORMs)",hold,"Languages and Frameworks",true,"Unless working with a large number of simple entities, prefer writing SQL directly or using a lightweight object mapper such as <a href='https://github.com/DapperLib/Dapper'>Dapper</a>."
-"Blazor.Net",assess,"Languages and Frameworks",true,"<a href='https://dotnet.microsoft.com/apps/aspnet/web-apps/blazor'>Blazor</a> is super exciting. It means we might get to leverage our existing C# skills in the browser. It's too early for Redgate to commit to using this in production, so it's in assess so we can keep a good understanding of developments in this area."
+"Blazor",hold,"Languages and Frameworks",true,"<a href='https://dotnet.microsoft.com/apps/aspnet/web-apps/blazor'>Blazor</a> is super exciting. It means we might get to leverage our existing C# skills in the browser. It's too early for Redgate to commit to using this in production, so it's in assess so we can keep a good understanding of developments in this area. <a href='https://github.com/red-gate/Tech-Radar/issues/361'>discussion to upgrade</a>"
```

Comment:
It's moved to Hold, but the comments state it's in Assess.

Reply:
🤦 (I'll fix that up!).